test/Microsoft.AspNetCore.Mvc.Analyzers.Test/TestFiles/AttributesShouldNotBeAppliedToPageModelAnalyzerTest/DiagnosticsAreReturned_IfFiltersAreAppliedToPageHandlerMethod.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Runtime.CompilerServices;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace Microsoft.AspNetCore.Mvc.Analyzers.Test
+{
+    public class DiagnosticsAreReturned_IfFiltersAreAppliedToPageHandlerMethod : PageModel
+    {
+        [/*MM*/ServiceFilter(typeof(object))]
```

Comment:
what is `/*MM*/`?

Reply:
It's a marker for where we expect the analyzer diagnostic to show up. This way the locations don't need to be redone in the event the code gets refactored. 

The test infrastructure parses these: https://github.com/aspnet/Common/blob/dev/src/Microsoft.AspNetCore.Analyzer.Testing/TestSource.cs#L12